src/components/message.vue

```diff
@@ -0,0 +1,588 @@
+<template>
+    <div class="comments_item">
+      <div class="comment_user_info">
+        <img
+          :src="
+            require(`../photos/photosForProfile/${commentAuthor.pathToPhoto}`)
+          "
+          :alt="commentAuthor.accountName"
+          width="32"
+          height="32"
+        />
+        <p class="comment_user_name">{{ commentAuthor.accountName }}</p>
+        <span v-if="isCurrentUserComment" class="its_you">you</span>
+        <p class="comment_user_date">{{ dateDifference(comment.date) }}</p>
+      </div>
+  
+      <textarea
+        v-if="isEditable"
+        v-model="textareaValue"
+        class="comment_user_text comment_user_edit_textarea"
+      />
+      <ReferenceChecking v-else :commentText="textareaValue" />
```

Comment:
preferred way to pass props to components in Vue is using kebab-case. these things are important because it makes easier to find things in code. e.g. if I wanted to find where `comment-text` prop is used I would search for it in kebab-case
```suggestion
      <ReferenceChecking v-else :comment-text="textareaValue" />
```